static/js/main.js

```diff
@@ -1,33 +1,59 @@
 "use strict";
 
-(function()
-{
-  function initLanguageSelection()
-  {
-    var locale = document.getElementById("navbar-locale-selected");
-
-    // skip if page does not have language selection (EG: blog)
-    if (!locale) return;
-
-    locale.addEventListener("click", function()
-    {
-      document.getElementById("navbar-locale-menu")
-        .classList.toggle("visible");
-    }, false);
-  }
-
-  function navigationClick(event)
-  {
-    document.getElementById("navbar-menu")
-      .classList.toggle("visible");
-  }
-
-  function initMenu()
-  {
-    document.getElementById("navbar-menu-toggle")
-      .addEventListener("click", navigationClick, false);
-  }
-
-  initLanguageSelection();
-  initMenu();
+(function() {
+    function initLanguageSelection() {
+        var locale = document.getElementById("navbar-locale-selected");
+
+        // skip if page does not have language selection (EG: blog)
+        if (!locale) return;
+
+        locale.addEventListener("click", function() {
+            document.getElementById("navbar-locale-menu")
+                .classList.toggle("visible");
+        }, false);
+    }
+
+    function navigationClick(event) {
+        document.getElementById("navbar-menu")
+            .classList.toggle("visible");
+    }
+
+    function initMenu() {
+        document.getElementById("navbar-menu-toggle")
+            .addEventListener("click", navigationClick, false);
+    }
+
+    function navbarToggle() {
+        var navBar = document.getElementById("navbar");
+        var navbarHeight = navBar.offsetHeight;
```

Comment:
According to MDN "This property will return null on Internet Explorer (9) if the style.position of the element itself is set to "fixed""... We may gracefully degrade (not support this feature without causing an error or other unexpected behaviour) this feature in IE9 if there is not another painless way to get offsetHeight across browsers including IE9.

This could be done by e.g. `if (!navbarHeight) return;`. (if we add a comment above explaining why this is necessary).